### ISSUE TITLE
hello - is this tool still maintained? thanks

### ISSUE THREAD

**Issue by ipstone** [kvch/wfw#6]

**Comment by kvch** [comment]
No, it is not maintained anymore. I will add a note about it to the readme.

**Comment by ipstone** [comment]
thanks for the reply - might be the reason that you are not using workflowy now, or there's some technical/setup change that require additional work to make this working with current workflowy website? 

thanks
